wgpu-core/src/command/mod.rs

```diff
@@ -9,8 +9,8 @@ mod render;
 mod transfer;
 
 pub use self::bundle::*;
-pub(crate) use self::clear::collect_zero_buffer_copies_for_clear_texture;
-pub use self::clear::ClearError;
+pub(crate) use self::clear::clear_texture_no_device;
+pub(crate) use self::clear::ClearError;
```

Comment:
lets merge these pub use statements per crate. I.e. `pub use self::{...};` or at least the 2 for clear :)